### PR TITLE
Issue 500a fix assets for deployed design consults site

### PR DESCRIPTION
**Pull request by lilaconlee**
Crux of the issue here was assets loaded in HTML needed the correct path once on GitHub pages, which this PR resolves with more changes than I expected.

Ran into some issues with `BASE_PATH` not being defined at some point during the build, but upgrading Astro fixed that. The upgrade introduced a new issue with built files (in our case `diff.md`) ending up in `dist/` as `diff/index.html`, but our static files were `production.html`. I changed the static file structure to match so when we can always use the same iframe src rather than trying to figure out what the file path should be based on the view.

Unrelated to this fix, I added a layout for the production placeholder. It was using default HTML styles which was not the best look. 

TL;DR
* Upgraded Astro
* Changed static file structure 
* Updated all the asset paths
* Created a layout for placeholder production view md

### CODE REVIEW

**Comment by mvogelgesang** [comment]
@lilaconlee this looks great, appreciate the thoughtful updates to README and other annotations!
